GeorgianMemenator/Scenes/CatTagsList/CatTagsListView.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CatTagsListView.swift
+//  GeorgianMemenator
+//
+//  Created by Giorgi Makadze on 09.01.2023.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+
+struct CatTagsListView: View {
+    var body: some View {
```

Comment:
You need property `@ObservedObject var viewModel: CatTagsListViewModel` here. Please check `RedditView`